ydb/library/yql/providers/dq/task_runner_actor/task_runner_actor.cpp

```diff
@@ -89,10 +120,69 @@ class TOutputChannelReader {
         return result;
     }
 
-private:
+    TOutputChannelReadResult ReadWithSpilling() {
+        int maxChunks = std::numeric_limits<int>::max();
+        bool changed = false;
+        bool isChanFinished = false;
+        i64 remain = ToPopSize;
+        ui32 dataSize = 0;
+        bool hasData = true;
+        TOutputChannelReadResult result;
+
+        if (remain == 0) {
+            // special case to WorkerActor
+            remain = 5<<20;
```

Comment:
Is it possible to use `5_MB` here? Like in this line https://github.com/ydb-platform/ydb/pull/612/files?diff=unified&w=0#diff-0755ee9481d9c53853e4dc79733dd98ddf730219572b1cf5501dac5f3e285e41R35